### PR TITLE
Adds Bing query engine (closes #65).

### LINKED ISSUE #65: Add Bing query engine

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @6a68

### CODE REVIEW

File: recommendation/search/query/bing.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import requests
+
+from recommendation import conf
+from recommendation.memorize import memorize
+from recommendation.search.query.base import BaseQueryEngine
+
+
+class BingQueryEngine(BaseQueryEngine):
+    """
+    Query engine that returns results from Yahoo's BOSS API.
```
Comment (pdehaan) [inline recommendation/search/query/bing.py]: Yahoo?

Comment (chuckharmston) [inline recommendation/search/query/bing.py]: Copypasta. Thanks for catching! :spaghetti:

**Comment by 6a68** [comment]
@chuckharmston lgtm, should we just delete the yahoo code at this point? feel free to merge and optionally delete the BOSS connector

**Comment by chuckharmston** [comment]
I'm going to keep Yahoo around for now, since the plan is apparently to keep riding those coattails as long as we are able.

Thanks for the review!

**Comment by pdehaan** [comment]
Filed #69 to track removing BOSS API code.
Looks like they're planning on pulling life support on BOSS on **March 31, 2016** (so we won't be riding those coattails for _too_ long).

**Pull request merged**